fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts

```diff
@@ -2677,6 +2680,17 @@ export class PseudoFWC {
             sysPage: -1,
             side: 'RIGHT',
         },
+        '0000280': // AUDIO SWITCHING KNOB
```

Comment:
This comment should reflect the exact ECAM message. The whole point of these comments is to be searchable.

Reply:
Done